packages/E2ETest/windows/ReactUWPTestApp/App.xaml.cs

```diff
@@ -44,8 +44,7 @@ public App()
             Microsoft.ReactNative.Managed.AutolinkedNativeModules.RegisterAutolinkedNativeModulePackages(PackageProviders); // Includes any autolinked modules
 
             PackageProviders.Add(new Microsoft.ReactNative.Managed.ReactPackageProvider());
-            PackageProviders.Add(new ReflectionReactPackageProvider<App>());
-            PackageProviders.Add(new TreeDumpLibrary.ReactPackageProvider());
```

Comment:
this seems to remove the tree dump library react package provider too, is this intentional? If so, how will tests work without it?